src/components/base/accordion/Accordion.module.css

```diff
@@ -14,23 +14,31 @@
   cursor: pointer;
 }
 
+.element .header:hover {
+  border-style: solid;
+  border-width: 1px 0 1px 0;
+  border-color: var(--blue);
+  margin: -1px 0 -1px 0;
+}
+
 .element .header p {
   margin-right: var(--pt2);
 }
 
 .firstelement {
   width: 10%;
+  padding-left: var(--pt2);
 }
 
 .textbox {
   display: grid;
-  grid-template-columns: 5fr 7fr 6fr;
+  grid-template-columns: 30% 38% 32%;
```

Comment:
Hvorfor var det nu der bruges procenter?

Var det noget med Accordion'en som også bruges i HelpMenu?

Kan vi snakke om den her? Jeg har et forslag til en måde vi kan undgå det her fremover :blush:

Reply:
Og en ting mere. Tager denne højde for Mobile? Og i så fald, hvordan håndterer den lange materialetyper?

Reply:
ignoreret